### PR TITLE
Add issue templates

### PR DESCRIPTION
**Pull request by ndhanushkodi**
Changes proposed in this PR:
- Add issue templates

How I've tested this PR:
By reading a small sample of issues to try to figure out what to ask in the template that would be useful.

How I expect reviewers to test this PR:
As people who have triaged lots of consul-k8s/consul-helm issues, please do offer suggestions about things you think would be helpful to reduce back and forth in issues.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/config.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+blank_issues_enabled: false
+contact_links:
+  - name: Consul-K8s Community Support
+    url: https://discuss.hashicorp.com/c/consul/29
+    about: If you have a question, or are looking for advice, please post on our Discuss forum! The community loves to chime in to help. Happy Coding!
+  - name: Consul-Helm GitHub Issues
+    url: https://github.com/hashicorp/consul-helm
```
Comment (ashwin-venkatesh) [inline .github/ISSUE_TEMPLATE/config.yml:7]: As in the other PR, link to issues perhaps?

**Review by ashwin-venkatesh — APPROVED** [review]
Similar suggestion wrt linking to issues page. i like the content here!!

**Review by lkysow — APPROVED** [review]
My comments from the helm pr apply here. Let's make sure we keep the templates in sync. Otherwise looks good!

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+### Reproduction Steps
+
+Steps to reproduce this issue, eg:
+
+1. When running consul-K8s command, eg:
+```
+consul-k8s \
+  inject-connect \
+  -default-inject=true
+```
```
[the remark this answers is not in the document]
Comment (ndhanushkodi) [reply .github/ISSUE_TEMPLATE/bug_report.md:24]: That sounds good! I added that to environment details.

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+output from 'kubectl logs'
+```
+
+</details>
+
+### Expected behavior
+
+What was the expected result?
+
+### Environment details
```
[the remark this answers is not in the document]
Comment (ndhanushkodi) [reply .github/ISSUE_TEMPLATE/bug_report.md:44]: Yeah, and I think I'm gonna add that to the consul helm bug report template as well

**Pull request merged**